schema_based_on_diagrame.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name TEXT,
+    date_of_birth
```

Comment:
- [ ] I think this field should have datatype, would you add it please? So that it won't throw you or put you in unexpected situation